doc/notes/cvar.txt

```diff
@@ -27,7 +27,7 @@ If the type is enum, the possible values of the enum need be listed in the descr
 Above example represents (in `src/include/mpir_cvars.h`):
     extern int MPIR_CVAR_IALLGATHER_INTRA_ALGORITHM;
     enum MPIR_CVAR_IALLGATHER_INTRA_ALGORITHM_choice {
-        MPIR_CVAR_IALLGATHER_INTRA_ALGORITHM_auto,
+        MPIR_CVAR_IALLGATHER_INTRA_ALGORITHM_json,
```

Comment:
so `_json` is the new `_auto` ?

Reply:
Basically, yes.  One difference is that the user can give a new json file at runtime (as a CVAR) to change the autoselection logic.